sdk/iOS/src/MSTableConnection.m

```diff
@@ -256,6 +256,39 @@ -(NSInteger) items:(NSArray **)items
     return totalCount;
 }
 
++(void) removeSystemColumnsFromItem:(NSMutableDictionary *)item ifNotInQuery:(NSString *)query
```

Comment:
This function should be gone in MobileApp, not sure why it got added again, can you drop this file's changes from yoru commit